BuildOpenCV.cmake

```diff
@@ -26,7 +26,14 @@ if (NOT EXISTS ${OPENCV_DOWNLOAD_DIR}/opencv-${OPENCV_VERSION}/CMakeLists.txt)
                 ${OPENCV_DOWNLOAD_DIR}/${OPENCV_VERSION}.zip
                 SHOW_PROGRESS
         )
-        message(STATUS "Downloading OpenCV - done")
+        message(STATUS "Downloading OpenCV-extra")
+        file(
+                DOWNLOAD
+                https://github.com/opencv/opencv_contrib/archive/refs/heads/4.x.zip
```

Comment:
Donwload the same version as the version of the main package. You can do this from tags, for example: `https://github.com/opencv/opencv_contrib/archive/refs/tags/${OPENCV_VERSION}.zip`

Reply:
Closing because this and all the conversions below as this change has been removed